channels/ba.m3u

```diff
@@ -9,13 +9,17 @@ https://bhrtstream.bhtelecom.ba/hls15/bhrtportal.m3u8
 https://dns2.rtvbn.com:8080/live/index.m3u8
 #EXTINF:-1 tvg-id="Malta.ba" tvg-country="BA" tvg-language="" tvg-logo="" group-title="",Malta (720p) [Offline]
 http://webtvstream.bhtelecom.ba/malta.m3u8
+#EXTINF:-1 tvg-id="ntvic.ba" tvg-country="BA" tvg-language="Bosnian" tvg-logo="https://i.imgur.com/Y4UJEfg.png" group-title="",NTV IC
+https://iptv.dc-team.com/query?url=https://www.twitch.tv/ntvic
 #EXTINF:-1 tvg-id="RTVBPK.ba" tvg-country="BA" tvg-language="Bosnian" tvg-logo="https://i.imgur.com/g639yyT.jpg" group-title="",RTV BPK (720p)
 https://webtvstream.bhtelecom.ba/gorazde_cam2.m3u8
 #EXTINF:-1 tvg-id="RTVHB.com" tvg-country="BA" tvg-language="Bosnian" tvg-logo="https://i.imgur.com/EE8C8rR.png" group-title="",RTV HB (576p)
 https://prd-hometv-live-open.spectar.tv/ERO_1_083/playlist.m3u8
 #EXTINF:-1 tvg-id="Televizija5.ba" tvg-country="BA" tvg-language="Bosnian" tvg-logo="https://i.imgur.com/hdwjkYa.png" group-title="",Televizija 5 (240p)
 https://balkanmedia.dynu.net/hls/tv5web.m3u8
-#EXTINF:-1 tvg-id="tvslon.ba" tvg-country="BA" tvg-language="Bosnian" tvg-logo="https://www.rtvslon.ba/wp-content/uploads/2020/07/slon-extra-logo-mali-300x277.jpg" group-title="",TV Slon Extra (1080p) [Offline]
-http://31.47.0.130:8082
```

Comment:
![image](https://user-images.githubusercontent.com/5430116/135716301-65d85d6c-5b0f-4cc1-ba57-4f2d2b13997b.png)
Connects and plays as fast as if it was my local stream

Reply:
same here.

Reply:
Same here.